src/news/release-8.4.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Prism Launcher Release 8.4, now available"
+description: Now with extra TNT!
+date: 2024-06-23
+release_version: "8.4"
+minimum_macos_version: 11.0.0
+macos_file_extension: zip
+macos_signature: u4nVbCdMLaPkIYma3VBONC9Nxwb+Nsad4fBqJ8Un9HcBCYSrRjrsja7A+cdWO/mrP8JUt6EyhGgLpGYDT2vZAA==
+tags:
+  - release
+---
+
+Hello there!
+
+We have accumulated a few changes and fixes in the past few weeks.
+The latest PrismLauncher update brings enhanced support for NeoForge and ATLauncher modpacks, along with significant improvements to user experience on macOS and critical bug fixes, including better CurseForge export and stability enhancements. Enjoy the new features and smoother operations with this release!
+
+Until next time!
+
+## Added
+
+- Add neoforge support for technic packs and atlauncher packs by [@Trial97](https://github.com/Trial97) in [#2382](https://github.com/PrismLauncher/PrismLauncher/pull/2382)
+- Add support for bcachefs for Copy-on-Write instance copies by [@KernelFreeze](https://github.com/KernelFreeze) in [#2438](https://github.com/PrismLauncher/PrismLauncher/pull/2438)
+- [macos] Add sdkman to java search paths  by [@dqfan2012](https://github.com/dqfan2012) in [#2377](https://github.com/PrismLauncher/PrismLauncher/pull/2377)
+
+## Changed
+
+- Update auth server ping with new domain by [@Trial97](https://github.com/Trial97) in [#2471](https://github.com/PrismLauncher/PrismLauncher/pull/2471)
+- [linux] shorten desktop entry comment field by [@deadmeu](https://github.com/deadmeu) in [#1032](https://github.com/PrismLauncher/PrismLauncher/pull/1032)
+- [macos] Improve blocked mods dialog by [@kthchew](https://github.com/kthchew) in [#2379](https://github.com/PrismLauncher/PrismLauncher/pull/2379)
+- improve invalid character removal from path by [@Trial97](https://github.com/Trial97) in [#2425](https://github.com/PrismLauncher/PrismLauncher/pull/2425) and [#2531](https://github.com/PrismLauncher/PrismLauncher/pull/2531)
```

Comment:
this definitely needs better naming

Reply:
I can't find the words to improve this, but any suggestion is welcome.
Is `Enhance the process of removing invalid characters from a path.` better?